docs/user-guide/FAQ.md

```diff
@@ -170,3 +170,9 @@ Screwdriver uses `sd_buildbot` as default [git user](https://github.com/screwdri
 This has implications for webhook processing. In order to prevent headless users from running your pipeline indefinitely, Screwdriver cluster admins can configure Screwdriver webhook processor to ignore commits made by headless users. This is done by setting [IGNORE_COMMITS_BY](https://github.com/screwdriver-cd/screwdriver/blob/ec959e1590909259479fe34f2f26d91f227025aa/config/custom-environment-variables.yaml#L323-L325) environment variable. Default git user `sd-buildbot` is usually added to this list.
 
 Users can override this behavior by using a different git user. For example `git config --global user.name my-buildbot`. With this, your `git` commits from a Screwdriver build will be associated with user `my-buildbot` and will run Screwdriver pipeline without getting ignored by webhook processor.
+
+## Why do my pull request builds fail with the error: `fatal: refusing to merge unrelated histories` in the sd-setup-scm step?
+
+If your pull request branch has more than `$GIT_SHALLOW_CLONE_DEPTH` commits (default: 50), you may see pull request builds fail with this error. This message indicates that git cannot find a common shared ancestor between your feature branch and the main branch.
+
+To resolve this issue, you could disable or tune the shallow clone settings ([see them listed here](./environment-variables#user-configurable)) for your job, or reduce the number of commits on your feature branch (e.g. rebase and squash).
```

Comment:
Of course, the other case is if you actually managed to create a pull request from a branch with no related history on purpose (or perhaps on accident). I'm not sure if Github would let you do this or not. 

If it would be useful I can mention that here, e.g.: 

> Screwdriver does not currently support pull request builds if your branches do not share a common base (e.g. the `git` command line option `--allow-unrelated-histories`).